packages/gatsby-plugin-image/README.md

```diff
@@ -71,55 +71,67 @@ Because the images still need to be resized during build, the props must be able
 
 This does not work:
 
-```js
+```jsx
 // ⚠️ Doesn't work
 
-({ logo }) => <Img src={logo}>
+import React from "react"
+import Img from "gatsby-image"
```

Comment:
This is not correct. The component is `StaticImage` from `"gatsby-plugin-image"`